libpod/container_internal_linux.go

```diff
@@ -1241,6 +1242,26 @@ func (c *Container) makeBindMounts() error {
 		c.state.BindMounts["/etc/hostname"] = hostnamePath
 	}
 
+	// Make /etc/localtime
+	if _, ok := c.state.BindMounts["/etc/localtime"]; !ok {
+		var zonePath string
+		if c.TZ() == "local" {
+			zonePath, err = filepath.EvalSymlinks("/etc/localtime")
+			if err != nil {
+				return errors.Wrapf(err, "error finding local timezone for container %s", c.ID())
+			}
+		} else {
+			zonePath = filepath.Join("/usr/share/zoneinfo", c.TZ())
+		}
+		localtimePath, err := c.copyTimezoneFile(zonePath)
+		if err == nil {
+			c.state.BindMounts["/etc/localtime"] = localtimePath
+		} else {
+			logrus.Warning("Official timezone not set, using custom")
```

Comment:
This is an error not a warning, return the error.
if err != nil {
 			return errors.Wriap(err, "Official timezone not set, using custom")
}